lib/Runtime/Library/BoundFunction.cpp

```diff
@@ -84,6 +94,12 @@ namespace Js
             // If no "this" is passed, "undefined" is used
             boundThis = scriptContext->GetLibrary()->GetUndefined();
         }
+
+        // Reduce length number of bound args
+        len = len - this->count;
+        len = max(len, 0);
+            
+        SetPropertyWithAttributes(PropertyIds::length, TaggedInt::ToVarUnchecked(len), PropertyConfigurable, nullptr, PropertyOperation_None, SideEffects_None);
     }
 
     BoundFunction::BoundFunction(RecyclableObject* targetFunction, Var boundThis, Var* args, uint argsCount, DynamicType * type)
```

Comment:
Haven't worked out the purpose of this alternate constructor - unsure if I need to add length handling in here as well.

Reply:
I suspect that this is dead code now; certainly chakracore doesn't reference it at all, and I also can't find any references in chakrafull either.